flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisConfigOptions.java

```diff
@@ -307,7 +307,16 @@ public class DorisConfigOptions {
                     .booleanType()
                     .defaultValue(false)
                     .withDescription("Whether to use buffer cache for breakpoint resume");
-
+    public static final ConfigOption<Boolean> USE_FLIGHT_SQL =
+            ConfigOptions.key("use_flight_sql")
+                    .booleanType()
+                    .defaultValue(Boolean.FALSE)
+                    .withDescription("USE_FLIGHT_SQL");
+    public static final ConfigOption<String> FLIGHT_SQL_PORT =
+            ConfigOptions.key("flight_sql_port")
+                    .stringType()
+                    .defaultValue("9040")
+                    .withDescription("FLIGHT_SQL_PORT");
```

Comment:
The style of configuration items can be unified, such as source.use-flight-sql
source.flight-sql-port